files/usr/share/cinnamon/cinnamon-settings/themes.ui

```diff
@@ -14,6 +14,8 @@
         <property name="halign">center</property>
         <property name="valign">start</property>
         <property name="margin-top">45</property>
+        <property name="margin-left">20</property>
+        <property name="margin-right">20</property>
```

Comment:
Can you use margin-start, -end instead of left/right (these are deprecated properties).